user_sync/connector/directory_ldap.py

```diff
@@ -324,6 +324,33 @@ def iter_search_result(self, base_dn, scope, filter_string, attributes):
                 connection.abandon(msgid)
             raise
 
+    @staticmethod
+    def format_ldap_query_string(query, **kwargs):
+        """
+        To be used with any string that will be injected into a LDAP query - this escapes a few special characters that
```

Comment:
Function comments need to be imperative, as in "Escape LDAP special characters that may appear in injected query strings."  Then put the usage info on the next line: "Should be used with any string that will be injected into a LDAP query."  Also it's "an LDAP" not "a LDAP" :).